test/models/members/group_member_test.rb

```diff
@@ -135,7 +135,7 @@ def setup
     group_and_ancestors = namespace.self_and_ancestors
     memberships = []
 
-    group_and_ancestors.each do |group|
```

Comment:
**suggestion:** This is for all the reminder of the ones in the test, all the places you changed to `map` it looks like they can all be simplified to something like

```ruby
files = sample.attachments.map { |attachment| attachment.file.blob }
```

Reply:
Fixed in 4785ce82cac4f161adeeec30a0e0b0a3ae7d0aef and 973b7d1c0bcb916fb0beec341cd6cf09dc473c60